data/hai/hai outfits.txt

```diff
@@ -797,6 +811,32 @@ outfit "Pulse Rifle"
 	"unplunderable" 1
 	description "The Pulse Rifle was first developed by the Hai to help protect their ships from boarders during the Korath raids. While it has since been adapted to more effectively incapacitate its target, if several bolts land in quick succession, the accumulated charge has a lethal effect on most species' nervous systems."
 
+outfit "Dual Pulse Pistols"
+	category "Hand to Hand"
+	plural "Dual Pulse Pistols"
+	cost 77777
+	thumbnail "outfit/dual pulse pistols"
+	"capture attack" 2.0
+	"capture defense" 1.4
+	"unplunderable" 1
+	description "A pair of modified heavy pulse pistols that are favored by some Unfettered and the occasional pirate captain. Dual wielding them adds more firepower during short raids against a ship's defenders. Individually, they lose their energy charge more quickly than a pulse rifle, making them a poor choice for holding a defensive position during a long siege."
+
+
+outfit "True Unfettered License"
+	category "Licenses"
+	thumbnail "outfit/true unfettered license"
+	description "This sword may be a bit of a strange license, but its intention is to inspire pride to the wearer, and when the Unfettered see it they show respect. This was given by one of their warlords as a sign of trust, and it is quite handy in fights."
+	description "	Somehow, the blade feels very light, and as though it was accompanying the strikes of its owner. The blade itself is mostly intended to be defensive, and fits you quite well even though it was clearly meant for a Hai."
+
+outfit "Ionic Blade"
+	category "Unique"
+	cost 1000000
+	thumbnail "outfit/ionic blade"
+	"capture attack" 10
+	"capture defense" 10
+	"energy consumption" 0.1
+	"unplunderable" 1
+	description "This legendary blade is able to affect the surrounding energy fields, powering down any energy-based weapons the enemy may have. The field can even sometimes be used to stun nearby enemies, or even a small section of their ships, in order to open hatches or temporarily deactivate defenses. However this blade needs to be recharged before each battles, and is made useless once its power supply runs out. It also has magnetic properties - the more charged it is, the more repulsive it is to any metals, making it a lifesaver in some scenarios. Some even say it can enact a small, temporary forcefield around its owner, in case of emergencies."
```

Comment:
```suggestion
	description "This legendary blade is able to affect the surrounding energy fields, powering down any energy-based weapons the enemy may have. The field can even sometimes be used to stun nearby enemies, or even a small section of their ships, in order to open hatches or temporarily deactivate defenses. However this blade needs to be recharged before each battle, and is made useless once its power supply runs out. It also has magnetic properties - the more charged it is, the more repulsive it is to any metals, making it a lifesaver in some scenarios. Some even say it can enact a small, temporary force field around its owner, in case of emergencies."
```